.github/workflows/build_and_push_latest_image.yml

```diff
@@ -32,7 +32,6 @@ jobs:
           QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
         run: |
           podman --version
-          cd Dockerfiles/midstream/
-          podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=master
+          podman build -t midstream_image:latest -f Dockerfiles/midstream/Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.9.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=master
```

Comment:
The tag `OPERATOR_TEST_PLAYBOOKS_TAG` is not used in any Dockerfile

We should rely on default value in Dockerfile for `OPERATOR_SDK_VERSION`
Otherwise people will need to update the same number on more places.
And people tend to forget :-) We should just remove it from github actions

Reply:
@samvarankashyap bump

Reply:
Updated.